docs/object-protection-testing.md

```diff
@@ -0,0 +1,428 @@
+### License
+
```

Comment:
Pls document here that data  and metadata FI is still WIP

Reply:
added